NEWS

```diff
@@ -28,33 +28,8 @@ $HEADER$
 
 ===========================================================================
 
-This file contains the main features as well as overviews of specific
-bug fixes (and other actions) for each version of Open MPI since
-version 1.0.
-
-As more fully described in the "Software Version Number" section in
-the README file, Open MPI typically releases two separate version
-series simultaneously.  Since these series have different goals and
-are semi-independent of each other, a single NEWS-worthy item may be
-introduced into different series at different times.  For example,
-feature F was introduced in the vA.B series at version vA.B.C, and was
-later introduced into the vX.Y series at vX.Y.Z.
-
-The first time feature F is released, the item will be listed in the
-vA.B.C section, denoted as:
-
-   (** also to appear: X.Y.Z) -- indicating that this item is also
-                                 likely to be included in future release
-                                 version vX.Y.Z.
-
-When vX.Y.Z is later released, the same NEWS-worthy item will also be
-included in the vX.Y.Z section and be denoted as:
-
-   (** also appeared: A.B.C)  -- indicating that this item was previously
-                                 included in release version vA.B.C.
-
-Master (not on release branches yet)
-------------------------------------
+This file contains the main features added, changed behavior, and legacy report
+removed for each version of Open MPI since version 1.0.
```

Comment:
This sentence is still... weird.  If you guys don't want to change it, fine.  But I'd suggest something like:

> This file contains the main features added, user-observable behaviors that were changed, and legacy functionalities that were removed for each version of Open MPI since version 1.0.

Reply:
Arrgh.  I commented on outdated text.  Ignore my prior remark (and will delete from github).